src/ssl.c

```diff
@@ -6647,9 +6802,18 @@ int ProcessFile(WOLFSSL_CTX* ctx, const char* fname, int format, int type,
             }
         }
         if ((type == CA_TYPE || type == TRUSTED_PEER_TYPE)
-                                              && format == WOLFSSL_FILETYPE_PEM)
+                                          && format == WOLFSSL_FILETYPE_PEM) {
             ret = ProcessChainBuffer(ctx, myBuffer, sz, format, type, ssl,
                                      verify);
+#ifndef NO_WOLFSSL_CM_VERIFY
+            /* Call to over-ride status */
+            if ((ctx != NULL) && (ctx->cm != NULL) &&
+                (ctx->cm->verifyCallback != NULL)) {
+                ret = wolfSSL_CertManagerVerifyBuffer_ex(ctx->cm, myBuffer, sz,
```

Comment:
This will also use the CM's verify callback for any CA or Trusted peer type certs when `ProcessFile` is called. I don't see this described in the PR comments or any test cases. Does this addition really need to be here?